bindings/python/tests/steps/binding.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Datafuse Labs.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from behave import given, when, then
+from behave.api.async_step import async_run_until_complete
+import _databend_driver
```

Comment:
why there is a prefix `_`?

Reply:
Maybe there is a problem but I can't find it out 😢 . I create the module named `databend_driver` but can't import it, only import `_databend_driver`. 

<img width="650" alt="image" src="https://github.com/datafuselabs/bendsql/assets/7600925/31df511a-f4cf-4b93-bc76-d50c00ef3666">

Reply:
ok, maybe I'll check it later.